### PR TITLE
bump deis to v1.4.1

### PR DESCRIPTION
**Pull request by bacongobbler**

### CODE REVIEW

File: Library/Formula/deis.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Deis < Formula
   homepage "http://deis.io"
-  url "https://github.com/deis/deis/archive/v1.4.0.tar.gz"
+  url "https://github.com/deis/deis/archive/v1.4.1.tar.gz"
   sha1 "96bb7bb71bfe32cfb5e148ccb9deabd621ddbcd4"
```
Comment (jacknagel) [inline Library/Formula/deis.rb]: checksum needs updating

Comment (bacongobbler) [inline Library/Formula/deis.rb]: missed that. Thanks! I'll fix that up

**Comment by jacknagel** [comment]
ping

**Comment by bacongobbler** [comment]
sorry, been away for the weekend. Will get to it tomorrow.

**Comment by asparagui** [comment]
Thanks!

**Pull request closed without merging**